gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri Jul 28 17:46:00 CEST 2017
+#Tue Aug 08 00:31:49 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.0.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
```

Comment:
`-all` contains source code - it is handy if you are debugging something. Try to import the project in idea, it will recommend it.